notification-config-service-api/src/main/proto/org/hypertrace/notification/config/service/v1/notification_rule.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.notification.config.service.v1;
+
+message NewNotificationRule {
+  string rule_name = 1;
+  string description = 2;
+  string environment = 3;
```

Comment:
environment isn't really relevant here. Should this support be moved into an event condition rather than first classing it?

Reply:
> environment isn't really relevant here. 
why?

tbh I just copied it from the previous proto, do you mean it should be in EventCondition proto in the other pr?